metrics/prometheus.go

```diff
@@ -186,6 +186,42 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 					}
 					return values
 				},
+			}, {
+				name:        "container_cpu_userspace_usage_seconds",
+				help:        "Per core time spent in user mode",
+				valueType:   prometheus.CounterValue,
+				extraLabels: []string{"cpu"},
+				getValues: func(s *info.ContainerStats) metricValues {
+					values := make(metricValues, 0, len(s.Cpu.Usage.PercpuUsageInUsermode))
+					for i, value := range s.Cpu.Usage.PercpuUsageInUsermode {
+						if value > 0 {
```

Comment:
In my opinion 0 value also gives some information and it's worth to have this metric so there is no need to create this case when value is higher than 0 .
Having this if-case requires additional information about this metric implementation and it's not user friendly.